src/pyproject_api/__main__.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import argparse
+import os
+import pathlib
+import sys
+import textwrap
+
+from ._frontend import EditableResult, SdistResult, WheelResult
+from ._via_fresh_subprocess import SubprocessFrontend
+
+
+def main_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        description=textwrap.indent(
+            textwrap.dedent(
+                """A pyproject.toml-based build frontend.
```

Comment:
Can we keep this one line please?

Reply:
We can, but is the rest of the context not helpful? If anything, I thought we'd want to expand it further.

Reply:
We can keep the words, just let the argparser break it up, no? Rather than us formatting it.

Reply:
Ohhh, gotcha :+1: Coming right up...